packages/components/src/ActionIntercom/Intercom.component.js

```diff
@@ -35,8 +35,8 @@ function Intercom({ id, className, config, t }) {
 	useLayoutEffect(() => IntercomService.setPosition(ref.current), [ref.current]);
 
 	const label = show
-		? t('TC_INTERCOM_CLOSE', { defaultValue: 'Close support messenger.' })
-		: t('TC_INTERCOM_OPEN', { defaultValue: 'Open support messenger.' });
+		? t('TC_INTERCOM_CLOSE', { defaultValue: 'Close chat with Talend support.' })
```

Comment:
Are you sure about the wording, I don't see the benefit of ending by a point?

Reply:
the wording was provided by PO